frontend/src/components/header/index.js

```diff
@@ -21,17 +22,46 @@ const menuItems = [
   {label: messages.help, link: "help"}
 ];
 
+
+const ActiveNavLink = (props) => {
+  const {isActive, ...otherProps} = props;
+  return(
+    <Link getProps={isActive} {...otherProps}>
+      {props.children}
+    </Link>
+  );
+};
+
+const UserDisplay = (props) => {
+  const imgClass = "br-100 h2 v-mid"
+  return(
+    <span>
+      {props.picture ?
+        <img className={imgClass} src={`https://www.gravatar.com/avatar/${props.picture}`}
+          alt={`${props.username} avatar`}
+        />
+        : <img className={imgClass} src={profilePic} alt={'user avatar'} />
+      }
+      <span className="pl 2">{props.username}</span>
+    </span>
+  );
+};
+
+
 class Header extends React.Component {
   menuItems = menuItems;
-  linkCombo = "link ph3 barlow-condensed blue-dark f4 ttu";
+  linkCombo = "link mh3 barlow-condensed blue-dark f4 ttu";
+  isActive = ({ isPartiallyCurrent }) => {
+    return isPartiallyCurrent ? {className: `${this.linkCombo} bb b--blue-dark bw1 pv2`} : {className: this.linkCombo};
+  }
 
   renderMenuItems() {
     return(
       <div className="v-mid">
         {this.menuItems.map((item, n) =>
-          <Link to={item.link} key={n} className={ this.linkCombo }>
-            <FormattedMessage {...item.label} />
-          </Link>
+          <ActiveNavLink to={item.link} key={n} isActive={this.isActive}>
+            <FormattedMessage {...item.label}/>
+          </ActiveNavLink>
```

Comment:
See above if you agree:
```suggestion
          </TopNavLink>
```